src/app/examples/crud/books.actions.ts

```diff
@@ -1,37 +1,19 @@
 import { Action } from '@ngrx/store';
-import { Update } from '@ngrx/entity';
-
 import { Book } from './books.model';
 
 export enum BookActionTypes {
-  ADD_ONE = '[Books] Add One',
-  UPDATE_ONE = '[Books] Update One',
-  DELETE_ONE = '[Books] Delete One',
-  SELECT_ONE = '[Books] Select One'
+  UPSERT_ONE = '[Books] Upsert One',
+  DELETE_ONE = '[Books] Delete One'
 }
 
-export class ActionBooksAddOne implements Action {
-  readonly type = BookActionTypes.ADD_ONE;
+export class ActionBooksUpsertOne implements Action {
```

Comment:
I'm fine with this change, but try to split it up in two PRs the next time 😄

Reply:
Okay, this change comes to help me workaround another issue 😅